lib/tbot/workloadidentity/workloadattest/sigstore/testutils.go

```diff
@@ -0,0 +1,88 @@
+package sigstore
```

Comment:
I'd be tempted to put this into a `sigstoretest` package or something to make this code obviously for testing only.